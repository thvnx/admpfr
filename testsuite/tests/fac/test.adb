with Ada.Text_IO; use Ada.Text_IO;
with Admpfr;      use Admpfr;

procedure Test is
   M : Mpfloat;
begin
   M.Fac (10);
   Put_Line (M'Image);
end Test;
