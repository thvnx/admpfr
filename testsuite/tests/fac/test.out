3.6288000000000000e+06
